### PR TITLE
feat(ui5-illustrated-message): add TNT illustration set

### PR DESCRIPTION
**Pull request by nnaydenow**

### CODE REVIEW

**Review by fifoosid — CHANGES_REQUESTED** [review]
You should fix the eslint

**Review by fifoosid — APPROVED** [review]

File: packages/tools/components-package/nps.js [hunk 1]
```diff
@@ -45,7 +49,7 @@ const getScripts = (options) => {
 				docs: `node "${LIB}/documentation/index.js" dist/api.json`,
 			},
 			illustrations: {
-				default: `node "${LIB}/create-illustrations/index.js" ${illustrationsPath} dist/illustrations`
```
Comment (vladitasev) [inline packages/tools/components-package/nps.js:48]: you don't need a `default` field if it's just one command, you can pass it directly to the `illustrations` property

**Review by vladitasev — COMMENTED** [review]
It's just small things, mostly naming

**Comment by nnaydenow** [comment]
#3613

**Pull request merged**